.gitignore

```diff
@@ -3,4 +3,6 @@ node_module
 .hintrc
 LICENSE
 style.css
-mobile.css
\ No newline at end of file
+mobile.css
+launch.json
+settings.json
```

Comment:
[OPTIONAL]
**Keep your repo clean and professional**
- Add external directories and files (``.vscode``) to your ``.gitignore`` file.

Reply:
Those files are for my working place and not related to repo at all. That's why I use gitignore to not pollute repo with unnecessary files. Or do you have other suggestions?